src/Events/Dispatcher.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Phalcon\Events;
+
+use Psr\EventDispatcher\EventDispatcherInterface;
+
+class Dispatcher implements EventDispatcherInterface
```

Comment:
This can be potentially implemented on EventManager making this class Not needed